### DISCUSSION TITLE
Is there a config option for disabling seamless playback

### DISCUSSION THREAD

**Discussion by TimVanDyke** [automatic-ripping-machine/automatic-ripping-machine#1172] [Q&A]
I would like to rip all the episodes individually (which is working fantastically) but then it also spits out a combined file for all episodes doubling the storage while in progress. (I generally rip an entire series, then write a handbrake command per series and do one huge batch of transcodes via a python script after the fact so I can tweak settings and experiment on the first episode to see what looks good)

Am I just missing how to do this?

**Comment by microtechno9000** [comment]
Disable main feature

**Reply by TimVanDyke** [reply]
I had it set to '0'

I'll try setting it to 'False'

Thanks!

**Reply by TimVanDyke** [reply]
It doesn't seem to have worked - it still created a file with all of them combined, any other ideas?
